CHANGELOG.md

```diff
@@ -13,7 +13,8 @@
 * Classes names "class_class_..." was not handled correctly in KeyPathMapping ([#4480](https://github.com/realm/realm-core/issues/4480))
 * Syncing large Decimal128 values will cause "Assertion failed: cx.w[1] == 0" ([#4519](https://github.com/realm/realm-core/issues/4519), since v10.0.0)
 * Fixed the query parser rejecting <,>,<=,>= queries on UUID types. ([#4475](https://github.com/realm/realm-core/issues/4475), since v10.0.0)
- 
```

Comment:
"Object builder" is a Java-ism and might not be clear to the other SDKs. Is it fair to say that this fixes an issue in `Object::create` and `Dictionary::insert` with dictionaries with null values?